src/actions/articles.js

```diff
@@ -12,3 +12,16 @@ export function getArticles() {
     });
   };
 }
+
+export function addArticle(articleDetails) {
+  return async (dispatch) => {
+    const response = await ArticlesApi.addArticle(articleDetails);
+    const json = await response.json();
+
+    if (json.status === 200) {
```

Comment:
Where are you getting the number 200 from, was it set from the API's side?

Reply:
200 is the HTTP code for OK. Check [the list](https://www.restapitutorial.com/httpstatuscodes.html) of the status codes. 404 is not found for example. They're standard and are returned form the BE.